internal/fsutil/create_test.go

```diff
@@ -73,18 +75,58 @@ func (s *S) TestCreate(c *C) {
 	}()
 
 	for _, test := range createTests {
+		if test.result == nil {
+			// Empty map for no files created.
+			test.result = make(map[string]string)
+		}
 		c.Logf("Options: %v", test.options)
 		dir := c.MkDir()
 		options := test.options
 		options.Path = filepath.Join(dir, options.Path)
-		err := fsutil.Create(&options)
+		fsCreator := fsutil.NewCreator()
+		err := fsCreator.Create(&options)
```

Comment:
Same as earlier, we can drop the "fs"/"FS" prefix here and below. Note how we are inside a "TestCreate" function with a "createTests" table. Let's please apply this throughout the PR.